client/homebrew/pages/userPage/userPage.less

```diff
@@ -30,4 +30,31 @@
 
 		}
 	}
+	.sort-container{
```

Comment:
Just minor comments on the styling:

* Vertical-align the text with the radio buttons if possible.
* "Open Sans" doesn't seem to be applying to the label items. See if you can get that to stick.
* Remove the border on the "dir" button and try to get the padding even all around its text.
* Change the arrow to something like U+1F83B or U+25BC to fit the blockier style of our other menu icons.
* "SORT TYPE:" to "SORT BY"